### PR TITLE
Configuration widget

### LINKED ISSUE #9006: Configuration Widget,  All steps in a page
- We move all steps in the same page.
- Delete the navbar (DATA/STYLE).
- Step 3 (Style) renamed to Behaviour.

Here a visual design

03 widgets - 01-00 histogram

<br>
cc/ @saleiva

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #9006.

widget-configuration

cc @xavijam

### CODE REVIEW

**Comment by xavijam** [comment]
Ok, so we will use the both form models but in the same "view", for me :+1:, please check in staging everything is ok 💋 .

**Comment by saleiva** [comment]
Hey, can we also remove the "UNFILTERED" option from everyplace?

**Comment by xavijam** [comment]
@nobuti, easy peasy and ping @csobier.

File: lib/assets/javascripts/cartodb3/editor/widgets/widgets-form/widgets-form-fields-view.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var _ = require('underscore');
 var Backbone = require('backbone');
 var CoreView = require('backbone/core-view');
 var WidgetFormFactory = require('./widgets-form-factory');
-var Template = require('./widgets-form-data.tpl');
+var Template = require('./widgets-form-fields.tpl');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widgets-form/widgets-form-fields-view.js]: I used to name templates in lowerCase due to they are not classes. Not sure if we should follow it, what do you think @CartoDB/frontend ?

Comment (matallo) [inline lib/assets/javascripts/cartodb3/editor/widgets/widgets-form/widgets-form-fields-view.js]: I've been seeing them both here and there, happy to stick to one, but no clear preference

Comment (nobuti) [inline lib/assets/javascripts/cartodb3/editor/widgets/widgets-form/widgets-form-fields-view.js]: For me is a variable, so I'd follow the same convention. In this case, I left it as it was 😞

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/widgets/widgets-form/widgets-form-fields-view.js]: Yeah, it was more for open a discussion, don't worry @nobuti

**Comment by xavijam** [comment]
Great change 💋  🇪🇸  🚢 🇮🇹

**Comment by nobuti** [comment]
Frontend test passing 🚀

**Comment by csobier** [comment]
Looks good!  "Behaviour" is the British English spelling though.  I would suggest "Behavior", as it is the more American English spelling

**Pull request merged**